projects/fsastorefrontlib/src/core/agent/facade/agent-search.service.ts

```diff
@@ -72,4 +73,12 @@ export class AgentSearchService {
       this.geolocationWatchId = null;
     }
   }
+
+  getResetSearchValue(): Observable<any> {
+    return this.resetSearchValue.asObservable();
+  }
+
+  setResetSearchValue(isReset: boolean): void {
+    return this.resetSearchValue.next(isReset);
+  }
```

Comment:
I would suggest to go with property here instead with method. There is no additional logic, it's just returning an observable. `checkout-config.service.ts` is a good example of the convention we should follow.